src/Repository/DefaultUnleashRepository.php

```diff
@@ -140,11 +140,10 @@ private function getCachedFeatures(): ?array
     {
         $cache = $this->configuration->getCache();
 
-        if (!$cache->has(CacheKey::FEATURES)) {
+        $result = $cache->get(CacheKey::FEATURES);
+        if ($result === null) {
```

Comment:
Could this $results from cache could have empty arrays like []?

Reply:
In the previous iteration yeah, in this, that shouldn't happen unless something puts an empty array in there. Assuming no bugs are present in the code, that should only ever happen if the SDK is using a token that's connected to a project that has no features